File: Java/opentraceability/src/main/java/opentraceability/mappers/EPCISDocumentMappers.java
package opentraceability.mappers;

import opentraceability.interfaces.*;
import opentraceability.mappers.epcis.json.*;
import opentraceability.mappers.epcis.xml.EPCISDocumentXMLMapper;
import opentraceability.mappers.masterdata.*;
import opentraceability.models.events.*;
import opentraceability.*;

public class EPCISDocumentMappers
{
	public IEPCISDocumentMapper XML = new EPCISDocumentXMLMapper();
	public IEPCISDocumentMapper JSON = new EPCISDocumentJsonMapper();
}

File: Java/OpenTraceability/src/main/kotlin/opentraceability/mappers/epcisdocumentmappers.kt
package opentraceability.mappers

import opentraceability.interfaces.IEPCISDocumentMapper
import opentraceability.mappers.epcis.json.EPCISDocumentJsonMapper
import opentraceability.mappers.epcis.xml.EPCISDocumentXMLMapper

class EPCISDocumentMappers {
    var XML: IEPCISDocumentMapper = EPCISDocumentXMLMapper()
    var JSON: IEPCISDocumentMapper = EPCISDocumentJsonMapper()
}
